meta-lxatac-software/recipes-support/chrony/files/chrony.conf

```diff
@@ -1,9 +1,9 @@
-# Use public NTP servers from the pool.ntp.org project.
-# Please consider joining the pool project if possible by running your own
-# server(s).
-# If you are a vendor distributing a product using chrony, you *MUST*
-# read and comply with http://www.pool.ntp.org/vendors.html
-pool 0.openembedded.pool.ntp.org iburst
+# Use public NTP servers from the pool.ntp.org project using the
+# Linux Automation GmbH vendor zone.
+# The NTP pool project provides public servers for time synchronization
+# consider joining the pool[1] if you are interested in having a great time.
+# [1]: https://www.pool.ntp.org/join.html
+pool 0.linux-automation.pool.ntp.org iburst
```

Comment:
We have got assigned multiple pools (with different hosts). One even has IPv6 addresses in it. Should we add more than just the first one here?

Reply:
I've updated the PR to include all of them